chapter2/tests/t20/terraform/base/rds.tf

```diff
@@ -0,0 +1,31 @@
+module "wordpress_mysql_rds" {
+  source                              = "terraform-aws-modules/rds/aws"
+  version                             = "5.9.0"
+  identifier                          = local.labels.rds
+  engine                              = var.rds_engine
+  family                              = var.rds_family
+  major_engine_version                = var.rds_major_engine_version
+  engine_version                      = var.rds_engine_version
+  instance_class                      = var.rds_instance_class
+  allocated_storage                   = var.rds_allocated_storage
+  max_allocated_storage               = var.rds_max_allocated_storage
+  skip_final_snapshot                 = true
+  db_name                             = var.rds_db_name
+  username                            = var.rds_db_user
+  create_random_password              = false
+  password                            = random_password.passwords_list["db_password"].result
+  port                                = var.rds_port
+  publicly_accessible                 = false
+  monitoring_interval                 = var.rds_monitoring_interval
+  backup_window                       = var.rds_backup_window
+  backup_retention_period             = var.rds_backup_retention_period
+  maintenance_window                  = var.rds_maintenance_window
+  vpc_security_group_ids              = [module.wordpress_rds_sg.security_group_id]
+  subnet_ids                          = toset(data.aws_subnets.rds.ids)
+  storage_encrypted                   = true
+  deletion_protection                 = false
+  options                             = []
+  create_db_parameter_group           = false
+  iam_database_authentication_enabled = true
+  enabled_cloudwatch_logs_exports     = ["audit", "error", "general", "slowquery"]
```

Comment:
should be coming from tfvars as they could become expensive.

So the hint - don't hardcode. ONLY hardcode what you mandatory to be this way for all envs and never change. For example `storage_encrypted=true` makes total sense, as it has no performance penalty or cost associated with it.

Reply:
> So the hint - don't hardcode. ONLY hardcode what you mandatory to be this way for all envs and never change. For example `storage_encrypted=true` makes total sense, as it has no performance penalty or cost associated with it.

@dmitry-mightydevops 
Yes, I understand, I just sometimes don't know if any value will ever change or not. For example, health_check in alb - should it be moved to .tfvars, or is it the same for all projects?
cloudwatch_logs - done

Reply:
the rule is simple
a. if you don't know - keep the defaults as defined by the module, i.e don't define this attr at the module level
b. if you know what this attr affects/or responsible for and it will deviate between envs and you will use it - extract into variables/auto.tfvars + env.tfvars